lib/logstash/pluginmanager/base.rb

```diff
@@ -0,0 +1,37 @@
+class LogStash::PluginManager::Base < Clamp::Command
+  def gemfile
+    @gemfile ||= LogStash::Gemfile.new(File.new(LogStash::Environment::GEMFILE_PATH, 'r+')).load
+  end
+
+  # If set in debug mode we will raise an exception and display the stacktrace
+  def report_exception(readable_message, exception)
+    if ENV["DEBUG"]
```

Comment:
it is kinda strange that the DEBUG flag changes the behaviour from printing message to raising exception.. is this necessary?